frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -34,85 +40,189 @@ interface SubscribedProps {
 type Props = SubscribedProps & AoppProps & TranslateProps;
 
 class Aopp extends Component<Props, State> {
-    private chooseAccount = (code: AccountCode) => {
-        aoppAPI.chooseAccount(code);
+    public readonly state: State = {
+        accountCode: '',
+    };
+
+    public componentDidMount() {
+        this.setAccountCodeDefault();
+    }
+
+    public componentDidUpdate(prevProps) {
+        if (this.props.aopp?.accounts !== prevProps.aopp?.accounts) {
```

Comment:
I don't think this check works, as two arrays are compared. I think it's only a reference comparison, not a value comparison. What about `const accountsKey = accounts => accounts.map(a => a.code).join()` and then compare `accountsKey(this.props.aopp.accounts) !== accountsKey(prevProps.aopp.accounts)` or similar?

Reply:
💯 , what do you think about using `utils/equal` https://github.com/digitalbitbox/bitbox-wallet-app/pull/1373/commits/78068e45f1a922c5e4d63abf189b15a9a3371320#diff-c4c5f0fa92cdc4f4a6fbdd048df4be6bea47bea9db5204b4656e7ac59ba96869R57